source/pigeonhole-man/option-d.inc.in

```diff
@@ -0,0 +1,9 @@
+**-d**
+   Don't write the binary to *out-file*, but write a textual dump of the
+   binary instead. In this context, the *out-file* value '-' has special
+   meaning: it causes the the textual dump to be written to **stdout**.
+   The *out-file* argument may also be omitted, which has the same
+   effect as '-'. The output is identical to what the
+   :man:`sieve-dump(1)` command produces for a compiled Sieve binary
+   file. Note that this option is not allowed when the *out-file*
+   argument is a directory.
```

Comment:
The -d argument is different between sievec and other sieve tools. This file suggests those were merged, which isn't possible.

Reply:
For most tools -d adds its own <dumpfile> parameter. For sievec, the optional second <outfile> argument is affected and -d does not add its own parameter.